src/TypeSpec.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -27,6 +26,7 @@ import {
     resolveOutputFolder
 } from "./options.js";
 import { Configuration } from "./type/configuration.js";
+import { stringifyRefs } from "./lib/json-serialize-refs.js";
```

Comment:
Can we find some other name instead of using  'json-serialize-refs' and 'stringifyRefs'

Reply:
do you have suggestions?

Reply:
If this is not a new implementation, I suggest follow the original license in our work and there is no need for naming update.